src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,26 @@
 package practice;
 
-public class CandidateValidator {
-    //write your code here
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+    private static final String PERSON_NATIONALITY = "Ukrainian";
+    private static final int INDEX_0 = 0;
+    private static final int INDEX_1 = 1;
```

Comment:
There is no any additional information in such naming. Why 0 and why 1 new person will never guess easily
```suggestion
    private static final int INDEX_DATE_FROM = 0;
    private static final int INDEX_DATE_TO = 1;
```